From 53a35e5b1aac29d6dac32b77721333a8be83718a Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.xyz>
Date: Mon, 20 Mar 2017 01:49:12 +0800
Subject: [PATCH 53/69] [DO NOT MERGE] arm64: allwinner: a64: enable RTL8211E
 PHY workaround

Some Pine64+ boards are said to have broken RTL8211E PHY.

Enable the workaround in Pine64+ device tree file.

Signed-off-by: Icenowy Zheng <icenowy@aosc.xyz>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pine64-plus.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pine64-plus.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pine64-plus.dts
index 7c19d11b1e7c..419b79f9761c 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pine64-plus.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pine64-plus.dts
@@ -54,3 +54,7 @@
 	pinctrl-0 = <&emac_rgmii_pins>;
 	phy-mode = "rgmii";
 };
+
+&ext_phy {
+	realtek,disable-rx-delay;
+};
-- 
2.12.2

